stagger/util.py

```diff
@@ -137,3 +137,8 @@ def print_warnings(filename, options):
                     print(filename + ":warning: " + str(w.message),
                           file=sys.stderr)
             sys.stderr.flush()
+
+def imghdr_what(file, h):
+    # imhdr.what was deprecated in PEP-0594
+    # See https://peps.python.org/pep-0594/#imghdr
+    return None
```

Comment:
This leaves 2 tests failing:

```
FAILED test/friendly.py::FriendlyTestCase::testPicture22 - AssertionError: 'Other(0)::<60511 bytes of None data>' != 'Other(0)::<60511...
FAILED test/friendly.py::FriendlyTestCase::testPicture23_24 - AssertionError: 'Other(0)::<60511 bytes of None data>' != 'Other(0)::<60511...
```

The full messages:

```
________________________ FriendlyTestCase.testPicture22 ________________________
[…]
E       AssertionError: 'Other(0)::<60511 bytes of None data>' != 'Other(0)::<60511 bytes of jpeg data>'
E       - Other(0)::<60511 bytes of None data>
E       ?                           ^^^
E       + Other(0)::<60511 bytes of jpeg data>
E       ?                           ^^ +

test/friendly.py:292: AssertionError
______________________ FriendlyTestCase.testPicture23_24 _______________________
[…]
E           AssertionError: 'Other(0)::<60511 bytes of None data>' != 'Other(0)::<60511 bytes of jpeg data>'
E           - Other(0)::<60511 bytes of None data>
E           ?                           ^^^
E           + Other(0)::<60511 bytes of jpeg data>
E           ?                           ^^ +

test/friendly.py:318: AssertionError
```

One solution is to provide a fallback implementation of `imghdr.what`, as I did in [macports/macports-ports 26385](https://github.com/macports/macports-ports/pull/26385/files#diff-1b431839f9106c8547a07d3092c701c970988c162ca8949fb4c16fc741967f2b).